libcontainer/process_linux.go

```diff
@@ -829,7 +829,13 @@ func (p *initProcess) startTime() (uint64, error) {
 	return stat.StartTime, err
 }
 
-func (p *initProcess) updateSpecState() error {
+func (p *initProcess) maybeUpdateSpecState() error {
```

Comment:
Minor nit: While there is only one caller of this and the semantics make sense for that caller, I think it'd be a bit easier to follow if the `if HasHook` check was in the caller and this method stayed as `updateSepcState`. As it stands, "maybe" doesn't tell me that much when I'm looking at the caller's code.

Reply:
I agree.

Even better, we can inline it since it's trivial and only has a single user.

Please see updated.